test/sqllogictest/dates-times.slt

```diff
@@ -217,6 +217,28 @@ SELECT DATE '2019-02-03' - DATE '2019-01-01';
 ----
 33
 
+# Date arithmetic with integer
```

Comment:
Can you add some adds and subs of i32::max to test overflow?

Reply:
Done